impl/src/main/java/com/okta/sdk/impl/ds/JacksonMapMarshaller.java

```diff
@@ -35,7 +43,13 @@
 
     public JacksonMapMarshaller() {
         this.objectMapper = new ObjectMapper();
+        this.objectMapper.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
         this.objectMapper.configure(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS, true);
+
+
+        SimpleModule simpleModule = new SimpleModule();
+        simpleModule.addSerializer(new AbstractResourceSerializer(AbstractResource.class));
```

Comment:
Supports AbstractResource maps that contain a 'map' instead of another AbstractResource